app/com/linkedin/drelephant/analysis/AnalyticJobGeneratorHadoop2.java

```diff
@@ -69,6 +64,8 @@
 
   private final Queue<AnalyticJob> _retryQueue = new ConcurrentLinkedQueue<AnalyticJob>();
```

Comment:
Can you change this to firstRetryQueue to maintain consistency?

Reply:
Done.